### DISCUSSION TITLE
Welcome to reasonml-idea-plugin Discussions!

### DISCUSSION THREAD

**Discussion by giraud** [giraud/reasonml-idea-plugin#298] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I will try to use discussions to store feature requests

**Comment by lloy0076** [comment]
Is the advice to "use another editor" (in the README.md) still active?

**Reply by giraud** [reply]
yes, because I can't promise anything if you use that editor.
